src/autoscaler/db/sqldb/scheduler_sqldb.go

```diff
@@ -82,3 +83,23 @@ func (sdb *SchedulerSQLDB) GetActiveSchedules() (map[string]*models.ActiveSchedu
 	return schedules, nil
 
 }
+
+func (sdb *SchedulerSQLDB) SynchronizeActiveSchedules(appIdMap map[string]bool) error {
+	if len(appIdMap) == 0 {
+		sdb.logger.Debug("No application exists")
+		return nil
+	}
+	var query string = "DELETE FROM app_scaling_active_schedule WHERE app_id NOT IN("
```

Comment:
I would rather we do this the right way and get the simple wins. It is easier to just collect the app guids from the active schedules and just determine the ones we need to remove.